src/main/java/racingcar/CarFactory.java

```diff
@@ -0,0 +1,24 @@
+package racingcar;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class CarFactory {
+	private static final String DUPLICATED_NAME_ERROR = "[ERROR] 차 이름은 중복될 수 없습니다.";
+
+	public static Set<Car> of(String names) {
+		String[] arrNames = names.split(",");
+		Set<Car> cars = Arrays.stream(arrNames)
+			.map(name -> new Car(name.trim()))
+			.collect(Collectors.toSet());
```

Comment:
```suggestion
		Set<Car> cars = Arrays.stream(arrNames)
		        .map(String::trim())
			.map(Car::new)
			.collect(Collectors.toSet());
```
가능하면 한번에 여러가지를 하지 말고 step by step으로 하나의 행위 위주로 작성해주는게 깔끔하답니다.

Reply:
좋은 팁 감사합니다!! 리뷰 반영했습니다!!